lib/stripe/core_resources/customer.ex

```diff
@@ -141,7 +141,7 @@ defmodule Stripe.Customer do
              } | %{}
   def list(params \\ %{}, opts \\ []) do
     new_request(opts)
-    |> prefix_expansions("data")
+    |> prefix_expansions()
```

Comment:
@swelham Are you ok with this change?

Reply:
Yes that looks good to me. I'm not sure why I didn't do it that way to start with 😕